scripts/metadata/definitions/unit.json

```diff
@@ -15,7 +15,7 @@
                  "type" : "enum",
                  "options": ["Ampere", "Kelvin", "Second"]
             },
-            "required" : true
+            "required" : false
```

Comment:
Any reason you turn all of them to "not required"?

Reply:
So that new instances don't require fields that are very specific when creating datasets and uploading files and might not apply to that community.